### DISCUSSION TITLE
Updating branding for a project in the "Projects using Symfony Components" page

### DISCUSSION THREAD

**Discussion by GuySartorelli** [symfony/symfony#51833] [Q&A] [answered]
Sorry if this is the wrong place for this, but I couldn't find an appropriate email address or form, so I figured I'd try this and at worst be directed to the correct contact method.

I'm a maintainer of Silverstripe CMS, which is listed on the [Projects using Symfony Components](https://symfony.com/projects) page, and on its own [child page](https://symfony.com/projects/silverstripe).

The branding on that page is outdated, and since we've recently added some [branding guidelines](https://www.silverstripe.com/silverstripe-brand-guides/silverstripe-cms-style-guide) for the project, I figure this is a good time to get that updated. Is there a way I can update that directly (e.g. via a pull request) somewhere?
If not, is there a process I can follow to get that updated?

**Comment by mbabker** [comment, marked as answer]
Send a pull request to https://github.com/symfony/marketing

**Reply by GuySartorelli** [reply]
Awesome, thank you.
